File: Easy/1051._Height_Checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        int count=0;
        vector<int> tmp=heights;
        sort(heights.begin(), heights.end());
        for(int i=0;i<tmp.size();i++){
            if(tmp[i]!=heights[i]) count++;
        }
        return count;
    }
};

File: Easy/1051._Height_Checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int count=0;
        int[] tmp=Arrays.copyOf(heights, heights.length);
        Arrays.sort(tmp);
        for(int i=0;i<heights.length;i++){
            if(heights[i]!=tmp[i]) count++;
        }
        return count;
    }
}
